src/OpenLoco/src/GameCommands/Vehicles/CreateVehicle.cpp

```diff
@@ -395,9 +394,9 @@ namespace OpenLoco::GameCommands
 
     static void sub_470312(VehicleHead* const newHead)
     {
-        _987C5C[_orderTableLength] = 0;
-        newHead->orderTableOffset = _orderTableLength;
-        _orderTableLength++;
+        _987C5C[OrderManager::numOrders()] = 0;
+        newHead->orderTableOffset = OrderManager::numOrders();
```

Comment:
Should really rename numOrders to orderTableLength but that's for another day.

Reply:
Happy to change numOrders to orderTableLength. Do you want that in a separate merge request?

Reply:
I think it's safe to do in this PR as well.

Reply:
Yeah second pr

Reply:
OK I'll go for a second pull request.
Will this pull request get merged, or do I need to do the "change numOrders to orderTableLength" pull request first and then amend this pull request?